src/main/resources/GoogleApps/Groovy.groovy

```diff
@@ -348,9 +349,10 @@ public class GoogleAppsGroovy {
             boolean foundCompatibleDomain = false;
             if(user.getEmailAddresses() != null) {
                 for(EmailAddress address: user.getEmailAddresses()) {
-                    String email = address.getValue();
-                    if(email.endsWith(DOMAIN)) {
+                    String emailA = address.getValue();
```

Comment:
``emailA`` is not a great name. Maybe ``userEmail``?

Reply:
Addressed. Both global and this one local.